packages/api/src/beacon/client/debug.ts

```diff
@@ -12,38 +16,55 @@ export function getClient(_config: IChainForkConfig, httpClient: IHttpClient): A
   const reqSerializers = getReqSerializers();
   const returnTypes = getReturnTypes();
   // Some routes return JSON, use a client auto-generator
-  const client = generateGenericJsonClient<Api, ReqTypes>(routesData, reqSerializers, returnTypes, httpClient);
+  const client = generateGenericJsonClient<Api, ReqTypes>(
+    routesData,
+    reqSerializers,
+    returnTypes as ReturnTypes<Api>,
+    httpClient
+  );
   // For `getState()` generate request serializer
   const fetchOptsSerializers = getFetchOptsSerializers<Api, ReqTypes>(routesData, reqSerializers);
 
   return {
     ...client,
 
+    // TODO: Debug the type issue
```

Comment:
Can you also think about how we would be able to incorporate this? https://github.com/ethereum/beacon-APIs/issues/250

Reply:
We already have support for SZZ encoding for few endpoints, similarly we can support for others when required. 

https://github.com/ethereum/beacon-APIs/blob/6669d981486b7034fc9911fdf7be157343b6ea5e/apis/debug/state.v2.yaml#L40-L42

Reply:
This beacon-apis PR is about systematically supporting SSZ for all endpoints, not just supporting one-by-one.